validator/client/validator_attest_test.go

```diff
@@ -130,7 +114,7 @@ func TestAttestToBlockHead_AttestsCorrectly(t *testing.T) {
 	validator.SubmitAttestation(context.Background(), 30, validatorPubKey)
 
 	aggregationBitfield := bitfield.NewBitlist(uint64(len(committee)))
-	aggregationBitfield.SetBitAt(4, true)
+	aggregationBitfield.SetBitAt(0, true)
```

Comment:
Why?

Reply:
before we hardcoded validator index request to always return 4th validator, we no longer request validator index hence the new validator index is the first validator in the committee given the pubkey we pass in